requests/transport.go

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (C) 2019 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package requests
+
+import (
+	"net"
+	"net/http"
+	"time"
+
+	"github.com/rs/zerolog/log"
+)
+
+const (
+	maxHTTPRetries  = 3
+	delayAfterRetry = 50 * time.Millisecond
```

Comment:
Does it enough to have only 150ms(3*50) for retries?

Reply:
I would say, in case we need a retry, ideally it should be exponential, but for the sake of simplicity we just should make it acceptable to the user. Lets make this (3*350).